CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ We use [semantic versioning](http://semver.org/):
 - [breaking change] removed options `coverage-via-http` and `teamscale-testwise-upload`. Use `tia-mode=http` or `tia-mode=teamscale-upload` instead.
 - [breaking change] _tia-client_: changed Java API
 - [feature] always exclude common libraries from profiling to shrink coverage files
+- [feature] enable changing and reading the commit message via a new `/message` REST endpoint
+- [breaking change] changing the partition is now a PUT request with the new partition in the request body (formerly POST with the new partition in the URL)
```

Comment:
The whole `with` description is a bit wordy. I think `/partition/{partitionName}` and `/partition` are more effective at communicating here.